src/firecracker/tests/test_cpu_templates_api.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Rather than doing an integration test in Rust, and then triggering it from pytest, why not just make it in pytest?

Reply:
There is a similar thread with @kalyazin, but I will summarize my take:

* Easier debugging with Rust workspace dependency.
* Smaller (or no) reliance on Docker environment updates with changing/updating dependencies.
* Existing Rust code is tested more thoroughly.
* Improves possibilities of clearer code coverage reporting.